src/ast/irbuilderbpf.h

```diff
@@ -78,6 +78,7 @@ class IRBuilderBPF : public IRBuilder<> {
                             Map &map,
                             Value *callback,
                             Value *callback_ctx,
+                            Value *return_ctx,
```

Comment:
Rather than an out-param, how about just making `CreateForEachMapElem()` return a `Value *`? `CallInst *`  is a valid `Value *`.